Author: Sergei Golovan
Description: Patch replaces external JavaScript resource by
 a local one.
Last-Modified: Fri, 24 May 2024 19:36:36 +0300

--- a/make/ex_doc.exs
+++ b/make/ex_doc.exs
@@ -187,7 +187,7 @@
   before_closing_body_tag: fn
     :html ->
       """
-      <script src="https://cdn.jsdelivr.net/npm/mermaid@10.9.0/dist/mermaid.min.js"></script>
+      <script src="/usr/share/nodejs/mermaid/dist/mermaid.js"></script>
       <script>
         document.addEventListener("DOMContentLoaded", function () {
           mermaid.initialize({
